package tt;

public class Trycatch {

	public static void main(String[] args) {
try {	int n[]= {1,2,3,4};
	
	
	System.out.println(n[2]);
	}
	catch(Exception e) {
		System.out.println("there is an exception");
	}
finally {
	System.out.println("thankyou");
}
	}	
	}


